integration-tests/pom.xml

```diff
@@ -52,6 +52,7 @@
         <module>neo4j</module>
         <module>mongodb-client</module>
         <module>resteasy-jackson</module>
+        <module>resteasy</module>
```

Comment:
Could we add these tests in the main ITs instead? We already have some RESTEasy tests there.